### ISSUE TITLE
[tool_crash] FileSystemException: Cannot delete file, OS Error: The process cannot access the file because it is being used by another process., errno = 32

### ISSUE THREAD

**Issue by Paramsaini** [flutter/flutter#70178]
## Command
```
flutter doctor --android-licenses
```

## Steps to Reproduce
1. ...
2. ...
3. ...

## Logs
FileSystemException: Cannot delete file, OS Error: The process cannot access the file because it is being used by another process.
, errno = 32
```
#0      _File.throwIfError (dart:io/file_impl.dart:635:7)
#1      _File._deleteSync (dart:io/file_impl.dart:298:5)
#2      FileSystemEntity.deleteSync (dart:io/file_system_entity.dart:455:47)
#3      ForwardingFileSystemEntity.deleteSync (package:file/src/forwarding/forwarding_file_system_entity.dart:72:16)
#4      ForwardingFileSystemEntity.deleteSync (package:file/src/forwarding/forwarding_file_system_entity.dart:72:16)
#5      GradleWrapper.updateInner.<anonymous closure> (package:flutter_tools/src/cache.dart:1045:70)
#6      _rootRunUnary (dart:async/zone.dart:1198:47)
#7      _CustomZone.runUnary (dart:async/zone.dart:1100:19)
#8      _FutureListener.handleValue (dart:async/future_impl.dart:143:18)
#9      Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:696:45)
#10     Future._propagateToListeners (dart:async/future_impl.dart:725:32)
#11     Future._completeWithValue (dart:async/future_impl.dart:529:5)
#12     _AsyncAwaitCompleter.complete (dart:async-patch/async_patch.dart:40:15)
#13     _completeOnAsyncReturn (dart:async-patch/async_patch.dart:311:13)
#14     ArtifactUpdater._downloadArchive (package:flutter_tools/src/cache.dart)
#15     _rootRunUnary (dart:async/zone.dart:1198:47)
#16     _CustomZone.runUnary (dart:async/zone.dart:1100:19)
#17     _FutureListener.handleValue (dart:async/future_impl.dart:143:18)
#18     Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:696:45)
#19     Future._propagateToListeners (dart:async/future_impl.dart:725:32)
#20     Future._completeWithValue (dart:async/future_impl.dart:529:5)
#21     Future._asyncCompleteWithValue.<anonymous closure> (dart:async/future_impl.dart:567:7)
#22     _rootRun (dart:async/zone.dart:1190:13)
#23     _CustomZone.run (dart:async/zone.dart:1093:19)
#24     _CustomZone.runGuarded (dart:async/zone.dart:997:7)
```
```
[✓] Flutter (Channel stable, 1.22.3, on Microsoft Windows [Version 10.0.19041.610], locale en-IN)
    • Flutter version 1.22.3 at C:\flutter
    • Framework revision 8874f21e79 (12 days ago), 2020-10-29 14:14:35 -0700
    • Engine revision a1440ca392
    • Dart version 2.10.3

[!] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
    • Android SDK at C:\Users\Paramjeet saini\AppData\Local\Android\sdk
    • Platform android-30, build-tools 30.0.2
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)
    ✗ Android license status unknown.
      Run `flutter doctor --android-licenses` to accept the SDK licenses.
      See https://flutter.dev/docs/get-started/install/windows#android-setup for more details.

[!] Android Studio (version 4.1.0)
    • Android Studio at C:\Program Files\Android\Android Studio
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)

[!] Connected device
    ! No devices available

! Doctor found issues in 3 categories.

```

## Flutter Application Metadata
No pubspec in working directory.

**Comment by hamdikahloun** [comment]
Hi @Paramsaini 

This is a duplicate of #69109 and a recent fix (PR #69115) has landed on master channel.

Can you please try on master?

**Comment by darshankawar** [comment]
@Paramsaini 
This issue was fixed and [merged](https://github.com/flutter/flutter/pull/69115). Please switch to latest master and try again to see if it works for you.
Closing this as fixed. If you disagree or still seeing the same issue on latest master, write in comments and I'll reopen it.
Thanks.
